src/sys/unix/tcp.rs

```diff
@@ -444,7 +447,15 @@ pub fn accept(listener: &net::TcpListener) -> io::Result<(net::TcpStream, Socket
     // But not all platforms have the `accept4(2)` call. Luckily BSD (derived)
     // OSes inherit the non-blocking flag from the listener, so we just have to
     // set `CLOEXEC`.
-    #[cfg(any(target_os = "ios", target_os = "macos", target_os = "solaris"))]
+    #[cfg(any(
+        all(
+            target_arch = "x86",
+            target_os = "android"
```

Comment:
Can you add a comment saying why we can't use `accept4(2)` on Android x86, also linking to https://github.com/tokio-rs/mio/issues/1445.

Reply:
Done! :)